core/doc_data.h

```diff
@@ -111,13 +181,67 @@ class DocData {
 		bool operator<(const ConstantDoc &p_const) const {
 			return name < p_const.name;
 		}
+		static ConstantDoc from_dict(const Dictionary &p_dict) {
+			ConstantDoc doc;
+
+			if (p_dict.has("name")) {
+				doc.name = p_dict["name"];
+			}
+
+			if (p_dict.has("value")) {
+				doc.value = p_dict["value"];
+			}
+
+			if (p_dict.has("is_value_valid")) {
+				doc.is_value_valid = p_dict["is_value_valid"];
+			}
+
+			if (p_dict.has("enumeration")) {
+				doc.enumeration = p_dict["enumeration"];
+			}
+
+			if (p_dict.has("is_bitfield")) {
+				doc.is_bitfield = p_dict["is_bitfield"];
+			}
+
+			if (p_dict.has("description")) {
+				doc.description = p_dict["description"];
+			}
+
+			if (p_dict.has("is_deprecated")) {
+				doc.is_deprecated = p_dict["is_deprecated"];
+			}
+
+			if (p_dict.has("is_experimental")) {
+				doc.is_experimental = p_dict["is_experimental"];
+			}
+
+			return doc;
+		}
 	};
 
 	struct EnumDoc {
 		String name = "@unnamed_enum";
 		bool is_bitfield = false;
 		String description;
 		Vector<DocData::ConstantDoc> values;
+		static ConstantDoc from_dict(const Dictionary &p_dict) {
+			ConstantDoc doc;
+
+			if (p_dict.has("name")) {
+				doc.name = p_dict["name"];
+			}
+
+			if (p_dict.has("is_bitfield")) {
+				doc.is_bitfield = p_dict["is_bitfield"];
+			}
+
+			if (p_dict.has("description")) {
+				doc.description = p_dict["description"];
+			}
+
```

Comment:
`values` missing ?

Reply:
Good catch ! 👍 

This is a copy/paste mistake that didn't popup during compilation given `EnumDoc` and `ConstantDoc` has similarly named field 😄